docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -77,6 +77,10 @@ If the [browsers](configuration-file.md#browsers) and [src](configuration-file.m
 > Important! Make sure to keep the browser tab that is running tests active. Do not minimize the browser window.
 > Inactive tabs and minimized browser windows switch to a lower resource consumption mode
 > where tests do not always execute correctly.
+>
+> Do not adjust page zoom during tests. Stable functionality on zoomed in our out pages is not guaranteed.
```

Comment:
Could you please also apply my remark about 'functionality' from the previous review:

https://github.com/DevExpress/testcafe/pull/5357#discussion_r460812053